aptos-move/move-examples/token_objects/sources/aptos_token.move

```diff
@@ -0,0 +1,841 @@
+    }
+
+    entry fun unfreeze_transfer_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+    ) acquires AptosToken {
+        unfreeze_transfer(creator, token_object(creator, &collection, &name));
+    }
+
+    entry fun set_description_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+        description: String,
+    ) acquires AptosToken {
+        set_description(creator, token_object(creator, &collection, &name), description);
+    }
+
+    entry fun set_name_call(
+        creator: &signer,
+        collection: String,
+        original_name: String,
+        new_name: String,
+    ) acquires AptosToken {
+        set_name(creator, token_object(creator, &collection, &original_name), new_name);
+    }
+
+    entry fun set_uri_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+        uri: String,
+    ) acquires AptosToken {
+        set_uri(creator, token_object(creator, &collection, &name), uri);
+    }
+
+    entry fun add_property_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+        key: String,
+        type: String,
+        value: vector<u8>,
+    ) acquires AptosToken {
+        let token = token_object(creator, &collection, &name);
+        add_property(creator, token, key, type, value);
+    }
+
+    entry fun add_typed_property_call<T: drop>(
+        creator: &signer,
+        collection: String,
+        name: String,
+        key: String,
+        value: T,
+    ) acquires AptosToken {
+        let token = token_object(creator, &collection, &name);
+        add_typed_property(creator, token, key, value);
+    }
+
+    entry fun remove_property_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+        key: String,
+    ) acquires AptosToken {
+        let token = token_object(creator, &collection, &name);
+        remove_property(creator, token, &key);
+    }
+
+    entry fun update_property_call(
+        creator: &signer,
+        collection: String,
+        name: String,
+        key: String,
+        type: String,
+        value: vector<u8>,
+    ) acquires AptosToken {
+        let token = token_object(creator, &collection, &name);
+        update_property(creator, token, &key, type, value);
+    }
+
+    entry fun update_typed_property_call<T: drop>(
+        creator: &signer,
+        collection: String,
+        name: String,
+        key: String,
+        value: T,
+    ) acquires AptosToken {
+        let token = token_object(creator, &collection, &name);
+        update_typed_property(creator, token, &key, value);
+    }
+
+    // Tests
+
+    #[test_only]
+    use std::string;
+    #[test_only]
+    use token_objects::collection;
+
+    #[test(creator = @0x123)]
+    entry fun test_create_and_transfer(creator: &signer) acquires AptosToken {
+        let collection_name = string::utf8(b"collection name");
```

Comment:
I was testing this the other day, and I don't think the function has to be entry to be a test

Reply:
yeah, we had breakage in behavior of entry